src/mongocxx/test/options/find_one_and_delete.cpp

```diff
@@ -30,10 +30,10 @@ TEST_CASE("find_one_and_delete", "[find_one_and_delete][option]") {
     options::find_one_and_delete opts{};
 
     std::chrono::milliseconds ms{400};
-    auto proj = document{} << "_id" << false << finalize;
+    auto projection = document{} << "_id" << false << finalize;
```

Comment:
Fair.  Note that I made this change since the variable name `projection` is used instead of `proj` everywhere else in the repository.